src/components/Accounts/Detail.tsx

```diff
@@ -73,7 +73,7 @@ const AccountDetail = () => {
             <AccountTransfers org={organization} txCount={transfersCount} />
           </TabPanel>
           <TabPanel>
-            <AccountFees org={organization} />
+            <AccountFees org={organization} feesCount={feesCount} />
```

Comment:
Why pass props from a hook? Directly use the hook in that component instead.

Reply:
I dont know why but it makes an additional call... let me double check

Reply:
Still, if you're already passing `organization` there's no need to add an additional prop you got from that `organization`.

Reply:
Additional calls are done, but i guess that is the normal behavior of react query.

I refactored to use the organization provider instead of pass organization data as an argument

67d3364

Reply:
About my previous comment I just realized that the call to `accounts` is done by the organization provider and not by the reactquery system. Maybe the lazyLoading of the tabs could be de cause ?